### ISSUE TITLE
Ability to use background image positioning & associated functions when assigning post featured image to background are not appearing in update 3.4

### ISSUE THREAD

**Issue by davetease1966** [elementor/elementor#16026]
After updating to Elementor 3.4 (Core) when adding a background image to a section via "post featured image" the functions of Position, Attachment, Repeat and Size are not available.

IF I add a specific image from the Media Library, however, the functions are there.

this seems to be affecting the additional breakpoints.

Screenshot
<img width="510" alt="Screen Shot 2021-08-17 at 10 35 41 am" src="https://user-images.githubusercontent.com/89053216/129807224-b74c9158-5dd7-415e-aa14-3a59334fd145.png">
 attached.

<img width="520" alt="Screen Shot 2021-08-18 at 8 11 45 am" src="https://user-images.githubusercontent.com/89053216/129807448-c0697b57-f1ad-487c-bfeb-d5c1492bd337.png">

## Prerequisites

- x I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- x The issue still exists against the latest stable version of Elementor.

## Isolating the problem

- x This bug happens with only Elementor plugin active (and Elementor Pro).
- x This bug happens with a default WordPress theme active.
- x I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
``` Running Hello Theme  - plain vanilla 

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.32-35
	PHP Version: 7.4.22
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.8
	Site URL: https://www.golfnsw.org.au
	Home URL: https://www.golfnsw.org.au
	WP Multisite: Yes
	Max Upload Size: 15 MB
	Memory limit: 768M
	Permalink Structure: /%category%/%year%/%monthnum%/%day%/%post_id%/%postname%/
	Language: en-AU
	Timezone: Australia/Sydney
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.4.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_AU
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Safari/605.1.15

== Active Plugins ==
	DynamicConditions
		Version: 1.5.1
		Author: RTO GmbH

	Ele Custom Skin
		Version: 3.1.2
		Author: Dudaster.com

	Elementor
		Version: 3.4.0
		Author: Elementor.com

	Elementor Blocks for Gutenberg
		Version: 1.0.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.3.7
		Author: Elementor.com

	Enable Media Replace
		Version: 3.5.0
		Author: ShortPixel

	Golf Genius Tournament Leaderboard Plugin
		Version: 1.0.0
		Author: Lyle Kirby

	Guest Author
		Version: 2.2
		Author: WebFactory Ltd

	Imagify
		Version: 1.10
		Author: WP Media

	JetBlocks For Elementor
		Version: 1.2.9
		Author: Crocoblock

	JetBlog For Elementor
		Version: 2.2.14
		Author: Crocoblock

	JetElements For Elementor
		Version: 2.5.8
		Author: Crocoblock

	JetEngine
		Version: 2.8.9
		Author: Crocoblock

	JetSearch For Elementor
		Version: 2.1.14
		Author: Crocoblock

	JetSmartFilters
		Version: 1.8.3
		Author: Crocoblock

	JetTabs For Elementor
		Version: 2.1.13
		Author: Crocoblock

	MonsterInsights - Google Optimize Addon
		Version: 1.3.2
		Author: MonsterInsights Team

	MonsterInsights - Page Insights Addon
		Version: 1.3.3
		Author: MonsterInsights Team

	MonsterInsights - Performance Addon
		Version: 1.5.0
		Author: MonsterInsights Team

	MonsterInsights Pro
		Version: 7.18.0
		Author: MonsterInsights

	Post Duplicator
		Version: 2.23
		Author: Metaphor Creations

	Post Tags and Categories for Pages
		Version: 1.4.1
		Author: curtismchale

	Post Type Switcher
		Version: 3.2.1
		Author: Triple J Software, Inc.

	ProProfs Embed
		Version: 1.0.0
		Author: ProProfs

	Query Monitor
		Version: 3.7.1
		Author: John Blackbourn

	Rank Math SEO
		Version: 1.0.70
		Author: Rank Math

	TablePress
		Version: 1.14
		Author: Tobias Bäthge

	TablePress Extension: Responsive Tables
		Version: 1.8
		Author: Tobias Bäthge

	Unlimited Elements for Elementor (Premium)
		Version: 1.4.80
		Author: Unlimited Elements

	Widget Disable
		Version: 2.0.0
		Author: required

	WP-ShowHide
		Version: 1.04
		Author: Lester 'GaMerZ' Chan

	WP Google Maps
		Version: 8.1.14
		Author: WP Google Maps

	WP Google Maps - Gold Add-on
		Version: 5.1.7
		Author: WP Google Maps

	WP Google Maps - Pro Add-on
		Version: 8.1.13
		Author: WP Google Maps

	WP Rocket
		Version: 3.9.1.1
		Author: WP Media

	WP Rocket | No Cache for Admins
		Version: 
		Author: WP Rocket Support Team

	YIKES Simple Taxonomy Ordering
		Version: 2.3.3
		Author: YIKES, Inc.


== Network Plugins ==
	Advanced Editor Tools (previously TinyMCE Advanced)
		Version: 5.6.0
		Author: Automattic

	Code Snippets
		Version: 2.14.1
		Author: Code Snippets Pro

	Golf Tournament Leaderboard Plugin
		Version: 1.0.0
		Author: Lyle Kirby

	OoohBoi Steroids for Elementor
		Version: 1.7.9
		Author: OoohBoi

	Plugins Garbage Collector (Database Cleanup)
		Version: 0.12
		Author: Vladimir Garagulya

	Rank Math SEO
		Version: 1.0.70
		Author: Rank Math

	Rank Math SEO PRO
		Version: 2.14.0
		Author: Rank Math

	Wordfence Security
		Version: 7.5.5
		Author: Wordfence

	WordPress Multisite Content Copier/Updater
		Version: 1.4.0
		Author: Obtain Infotech


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatibility not specified
	JetBlocks For Elementor: Compatibility not specified
	JetBlog For Elementor: Compatibility not specified
	JetElements For Elementor: Compatibility not specified
	JetSearch For Elementor: Compatibility not specified
	JetTabs For Elementor: Compatibility not specified
	Unlimited Elements for Elementor (Premium): Compatibility not specified

== Elementor Pro - Compatibility Tag ==
	


== Elements Usage ==
	
	loop : 6
		call-to-action : 2
		image-box : 1
	wp-post : 180
		accordion : 1
		button : 2
		heading : 3
		icon-list : 1
		share-buttons : 1
		text-editor : 3
	error-404 : 1
		heading : 3
		search-form : 1


== Log ==
	
PHP: showing 2 of 2PHP: 2021-08-18 07:30:06 [error X 11][/home/customer/www/golfnsw.org.au/public_html/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php::180] Uncaught Error: Call to a member function get_component() on null in /home/customer/www/golfnsw.org.au/public_html/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php:180
Stack trace:
#0 /home/customer/www/golfnsw.org.au/public_html/wp-content/plugins/elementor-pro/modules/global-widget/widgets/global-widget.php(68): ElementorPro\Modules\GlobalWidget\Widgets\Global_Widget->set_preview_settings(Array)
#1 /home/customer/www/golfnsw.org.au/public_html/wp-content/plugins/elementor/includes/base/element-base.php(537): ElementorPro\Modules\GlobalWidget\Widgets\Global_Widget->get_raw_data(false)
#2 /home/customer/www/golfnsw.org.au/public_html/wp-content/plugins/elementor/includes/base/element-base.php(537): Elementor\Element_Base->get_raw_data(false)
#3 /home/customer/www/golfnsw.org.au/public_html/wp-content/plugins/elementor/core/base/document.php(834): Elementor\Element_Base->get_raw_data(false)
#4 /home/customer/www/golfnsw.org.au/public_html/wp-content/plugins/elementor-pro/modules [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-18 08:00:14 [notice X 1][/home/customer/www/golfnsw.org.au/public_html/wp-content/plugins/elementor-pro/modules/forms/classes/form-record.php::229] Undefined index: field_type [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 402021-08-18 08:00:01 [info] elementor-pro::elementor_pro_updater Started 
2021-08-18 08:00:01 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-08-18 08:00:01 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-08-18 08:00:01 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2021-08-18 08:04:59 [info] elementor-pro::elementor_pro_updater Started 
2021-08-18 08:04:59 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-08-18 08:05:00 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-08-18 08:05:00 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2021-08-18 08:10:06 [info] elementor-pro::elementor_pro_updater Started 
2021-08-18 08:10:06 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-08-18 08:10:06 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-08-18 08:10:06 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2021-08-18 08:14:59 [info] elementor-pro::elementor_pro_updater Started 
2021-08-18 08:14:59 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-08-18 08:14:59 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-08-18 08:14:59 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  
2021-08-18 08:20:00 [info] elementor-pro::elementor_pro_updater Started 
2021-08-18 08:20:00 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Start  
2021-08-18 08:20:01 [info] Elementor Pro/Upgrades - _v_3_3_0_nav_menu_icon Finished 
2021-08-18 08:20:01 [info] Elementor Pro/Upgrades - _v_3_3_0_recalc_usage_data Start  


				
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by shilo-ey** [comment]
I merged this issue with #16003, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!

**Comment by davetease1966** [comment]
updated to 3.4.1 - not fixed
